src/staticdata_utils.c

```diff
@@ -510,7 +510,7 @@ static void jl_collect_edges(jl_array_t *edges, jl_array_t *ext_targets, jl_arra
 #ifndef NDEBUG
                     jl_methtable_t *mt = jl_method_get_table(m);
                     if ((jl_value_t*)mt != jl_nothing) {
-                        matches = jl_gf_invoke_lookup_worlds(invokeTypes, (jl_value_t*)mt, world, &min_valid, &max_valid);
+                        jl_value_t *matches = jl_gf_invoke_lookup_worlds(invokeTypes, (jl_value_t*)mt, world, &min_valid, &max_valid);
```

Comment:
Maybe give it another name if it's not the same thing as the `matches` above?

Reply:
It is the same, we just are using it to assert that (note the NDEBUG above it)

Reply:
Oh, I missed that oops